### DISCUSSION TITLE
Backing up player backpack inventory/configurations?

### DISCUSSION THREAD

**Discussion by pokemongojsb1** [P3pp3rF1y/SophisticatedBackpacks#596] [General]
Is there a way to do a backup of player's backpacks on my server?

For builds, I use WorldEdit to make a schematic of the players buildings.   If a backpack is present, it comes out empty on the LOAD/RELOAD, so all items are lost.

Is there a way to back up the backpack data of a player?

**Comment by P3pp3rF1y** [comment]
Backpack contents are stored in `sophisticatedbackpacks.dat` so you would need to copy that to the new world.

**Reply by xylewis** [reply]
Does this file `sophisticatedbackpacks.dat` store all the backpack data in this A world? If it is directly copied to B world, will it overwrite the backpack data in  B world?
